bs-react-native-next/src/Style.re

```diff
@@ -252,6 +252,12 @@ external style:
   t =
   "";
 
-// Duplicated from StyleSheet.re for convenience.
-[@bs.module "react-native"] [@bs.scope "StyleSheet"]
+[@deprecated "Use Style.array([|t|]) instead"]
+[@bs.module "react-native"]
+[@bs.scope "StyleSheet"]
 external flatten: array(t) => t = "";
```

Comment:
I think we can just remove this instead of deprecating it as -next is still WIP/unreleased anyway.

Reply:
Done.